tests/Cli/PhpsuApplicationTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace PHPSu\Tests\Cli;
+
+use PHPSu\Cli\PhpsuApplication;
+use PHPSu\Cli\SshCliCommand;
+use PHPSu\Cli\SyncCliCommand;
+use PHPUnit\Framework\TestCase;
+
+class PhpsuApplicationTest extends TestCase
+{
+    public function testCommand()
+    {
+        $app = PhpsuApplication::createApplication();
```

Comment:
so, command basically is going to be useless in the future?

Reply:
why?

Reply:
because it currently is calling just another method

Reply:
it calls 2 methods